pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 1.3.1
```

Comment:
@ekiernan Would you mind updating the version in the Multi-snSS2 Overview and Methods docs as well?